### ISSUE TITLE
AVT 1 - Skeleton: Accordion & Button  DAP violation for missing form control label

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4177]
##Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

##Detailed Description
Go to the React skeleton for Accordion & Button
Run DAP and the following issue is found on the on each skeleton component (see screenshots):
<img width="1419" alt="Accordion" src="https://user-images.githubusercontent.com/21676914/66067398-1b7ea480-e511-11e9-9632-6199cf49507d.png">
